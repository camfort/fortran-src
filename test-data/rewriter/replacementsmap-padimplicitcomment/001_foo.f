      program main
      integer hello*4 ! This is a long comment that goes over the 72 columns
     +      , hello2*2
      end program main