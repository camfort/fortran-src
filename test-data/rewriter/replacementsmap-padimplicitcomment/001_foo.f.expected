      program main
      ! This is a long comment that goes over the 72 columns
      integer*4 hello
      integer*2 hello2
      end program main